dsc/reports/finalize.py

```diff
@@ -75,19 +68,27 @@ def create_attachments(self) -> list[tuple]:
             )
         return attachments
 
-    def _write_processed_items_csv(self) -> StringIO:
-        """Write processed items to string buffer.
+    def get_ingested_items(self) -> list[dict]:
```

Comment:
I dig it! Nice and explicit.  This basically codifies than an "ingested" item is a "processed" item where the SQS message indicates "success".